llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -463,6 +528,22 @@ static string_vector saveDeviceImageProperty(
           {"isEsimdImage", true});
     }
 
+    {
+      Module *M = ResultModules[I].get();
+      std::vector<Function *> SyclKernels;
```

Comment:
What is a purpose of this vector?

Reply:
This vector is useless, thanks for pointing - I removed it in d860652